kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -67,11 +67,13 @@ class KyuubiSessionImpl(
   override def open(): Unit = {
     MetricsSystem.tracing { ms =>
       ms.incCount(CONN_TOTAL)
-      ms.incCount(MetricRegistry.name(CONN_OPEN, user))
     }
     withZkClient(sessionConf) { zkClient =>
       val (host, port) = engine.getOrCreate(zkClient)
       openSession(host, port)
+      MetricsSystem.tracing { ms =>
+        ms.incCount(MetricRegistry.name(CONN_OPEN, user))
+      }
```

Comment:
MetricsSystem.tracing(_.incCount(MetricRegistry.name(CONN_OPEN, user)))

Reply:
Sorry I have't seen this comment before merging, seems not a big deal